swagger_spec_validator/validator20.py

```diff
@@ -31,7 +32,7 @@
 log = logging.getLogger(__name__)
 
 
-def validate_ref(ref_dict):
+def validate_ref(ref_dict, path):
     """Check if a ref_dict has siblings that will be overwritten by $ref. Raise
     a SwaggerValidationError if extra items are found in ref_dict.
```

Comment:
Let's update the docstring too - we're not raising a SwaggerValidationError, and we're warning about None values too now.